isolate.sh

```diff
@@ -277,12 +277,16 @@ function update_tenant() {
         updated_yaml=$(echo "$current_yaml" | "${YQ}" eval 'with(.namespaceMapping; . += [{"map-to-common-service-namespace": "'$map_to_cs_ns'"}])')
     fi
 
-    local tmp="\"$map_to_cs_ns\","
+    local tmp="\"$map_to_cs_ns\""
     debug1 "map $map_to_cs_ns namespace $namespaces tmp $tmp"
     for ns in $namespaces; do
-        tmp="$tmp\"$ns\","
+        debug1 "ns $ns mapto: $map_to_cs_ns"
+        if [[ $ns != $map_to_cs_ns ]]; then
```

Comment:
Arent't there missing quotation marks around $ns and $map_to_cs_ns?

Reply:
Ben helps to address this in PR: https://github.com/IBM/ibm-common-service-operator/pull/1302/files
@bluzarraga